src/Symfony/Component/ClassLoader/ApcUniversalClassLoader.php

```diff
@@ -60,7 +60,7 @@
  *
  * @api
  */
-class ApcUniversalClassLoader extends UniversalClassLoader
+class ApcUniversalClassLoader extends ClassLoader
```

Comment:
This is wrong. The ApcUniversalClassLoader should be removed entirely (we already have the ApcClassLoader as a replacement, which is way better since it uses composition rather than inheritance).